### ISSUE TITLE
Lack of RadioNodeList

### ISSUE THREAD

**Issue by TechQuery** [jsdom/jsdom#2600]
## Basic info

- **Node.js version:** 8.0.0+
- **jsdom version:** 12.0.0+

## Minimal reproduction case

```js
const { JSDOM } = require("jsdom");

const { window: { document } } = new JSDOM(`
<form>
    <input type="radio" name="radio" value="r1">
    <input type="radio" name="radio" value="r2">

    <input type="checkbox" name="check" value="c1">
    <input type="checkbox" name="check" value="c1">
</form>
`);

const { radio, check } = document.form[0].elements;

console.assert(radio.constructor.name === 'RadioNodeList', 'List of Radio inputs');
console.assert(check.constructor.name === 'RadioNodeList', 'List of CheckBox inputs'); 
```

## Specification

https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#radionodelist

**Comment by alex-cleo** [comment]
Hello there.

Any updates on this? My code depends on `if (element instanceof RadioNodeList) { ... }` which makes the test fail.

**Comment by domenic** [comment]
https://mobile.twitter.com/slicknet/status/782274190451671040

**Comment by alex-cleo** [comment]
> 

I'll take a look after work. If I'm able to do it myself I'll gladly create one

**Comment by vlakoff** [comment]
This issue seems to have been fixed by #3337.

**Comment by TechQuery** [comment]
> This issue seems to have been fixed by #3337.

@vlakoff Thanks, @UndefinedBehaviour referred this issue instead of `fix` it, so it keeps open for 3 years...
